helix-rest/src/test/java/org/apache/helix/rest/server/TestClusterAccessor.java

```diff
@@ -405,7 +407,68 @@ public void testEnableDisableMaintenanceModeWithCustomFields() {
         accessor.getBaseDataAccessor().exists(accessor.keyBuilder().maintenance().getPath(), 0));
   }
 
-  @Test()
+  @Test(dependsOnMethods = "testEnableDisableMaintenanceModeWithCustomFields")
+  public void testGetStateModelDef() throws IOException {
+
+    System.out.println("Start test :" + TestHelper.getTestMethodName());
```

Comment:
It would be good to move the space to make it as "Start test: "?

Reply:
This makes sense to me. On the other hand, other tests also use "Start test :", with this extra space. Let us just keep it the same.